docker-compose/postgres-10_repmgr-3.2_pgpool-3.3_barman-2.3.yml

```diff
@@ -1,7 +1,7 @@
 
 ##########################################################################
 ##                         AUTO-GENERATED FILE                          ##
-##               BUILD_NUMBER=Sat 27 Apr 2019 12:29:35 +08              ##
+##               BUILD_NUMBER=Wed Jul  3 10:29:46 JST 2019              ##
```

Comment:
Do we really want to commit generated files with just a timestamp change?

Reply:
If we can stop doing it - would be nice.
Just keep in mind this build number is used for https://github.com/paunin/PostDock/blob/master/make/make.sh#L11